src/components/Features/Faq/Faq.text.js

```diff
@@ -0,0 +1,145 @@
+export const faqs = [
```

Comment:
Move the file under `config` dir and call it `faqs.js`

Reply:
how about 'faqs.js'?